src/app/zones/locales/en-us/index.yaml

```diff
@@ -1,29 +1,47 @@
 zone-cps:
   routes:
     item:
-      title: "{name} Zone CP"
-      breadcrumbs: Zone CPs
+      title: "{name} Control Plane"
+      breadcrumbs: Control Planes
       config:
         no-subscriptions: This zone has no subscriptions
     items:
-      title: Zone CPs
-      breadcrumbs: Zone CPs
+      title: Control Planes
+      breadcrumbs: Control Planes
+      headers:
+        name: 'Name'
+        zoneCpVersion: 'Zone CP Version'
+        storeType: 'Storage type'
```

Comment:
Please can you stop using camelCasing for i18n keys, I would imagine you can change these to be

```yaml
  version:
#and
  type:
#or
  storage:
```

Reply:
I’ve changed this to be lowercase. I won’t address anymore nitpicks related to message key casing.